### PR TITLE
Ensure sales channel updates don't remove sales channel on other products

### PR DESCRIPTION
**Pull request by sradevski**

### CODE REVIEW

File: packages/core/core-flows/src/product/workflows/update-products.ts [hunk 1]
```diff
@@ -38,6 +38,7 @@ function prepareUpdateProductInput({
 }: {
   input: WorkflowInput
 }): UpdateProductsStepInput {
+  console.log("INPUT", input)
```
Comment (adrien2p) [inline packages/core/core-flows/src/product/workflows/update-products.ts]: Todo rm

File: packages/core/core-flows/src/product/workflows/update-products.ts [hunk 2]
```diff
@@ -81,6 +83,8 @@ function prepareSalesChannelLinks({
   updatedProducts: ProductTypes.ProductDTO[]
   input: WorkflowInput
 }): Record<string, Record<string, any>>[] {
+  console.log("TO CREATE", input, updatedProducts)
```
Comment (adrien2p) [inline packages/core/core-flows/src/product/workflows/update-products.ts]: Todo rm

**Review by adrien2p — COMMENTED** [review]
Why did we remove all inventory_quantity? It seems unrelated to the pr scope no?

File: packages/core/core-flows/src/product/workflows/update-products.ts (hunk 2 above)
Comment (sradevski) [reply packages/core/core-flows/src/product/workflows/update-products.ts]: @adrien2p gosh :/ done

**Comment by sradevski** [comment]
@adrien2p they were already removed everywhere else, it's just that they were not removed in the api tests. The tests were failing because of that.

**Review by adrien2p — APPROVED** [review]
It seems to be good for me 👍

**Review by carlos-r-l-rodrigues — APPROVED** [review]

**Review by olivermrbl — APPROVED** [review]

**Pull request merged**